sootup.core/src/main/java/sootup/core/model/Body.java

```diff
@@ -141,9 +132,9 @@ private void runValidation(BodyValidator validator) {
   /** Verifies that each use in this Body has a def. */
   // TODO: #535 implement validator public void validateUses() {  runValidation(new
   // UsesValidator()); }
-  private void checkInit() {
-    runValidation(new CheckInitValidator());
-  }
+  //  private void checkInit() {
+  //    runValidation(new CheckInitValidator(),);
+  //  }
```

Comment:
I think it's better not to comment these out (on lines 91 and  273 too), but do nothing inside the validators.

Reply:
Done!